src/test_similarities.py

```diff
@@ -0,0 +1,118 @@
+import os
+
+from compute_embedding import compute_similarity_of_texts
+
+def check_cosine_similarity(filename_in: str, filename_out: str) -> None:
+    """This function reads in a file line by line and splits each line into
+    a list. Each line should be split at the space character and should contain
+    five strings. The first four strings are filenames of text files which have
+    to be stored in /data/testTextPairs_WebForm. The cosine similarity of the
+    texts in the first two files and the cosine similarity of the texts in the
+    last two files is computed. The similarities of the files are written to
+    filename_out. It is tested which cosine similarity is higher or rather the
+    content of which of the files is more similar. The result is written to
+    filename_out. The fifth string in each line is a boolean value which
+    indicates whether the first two files are more similar or the last two
+    files. A '1' indicates that the texts of the last two files are more
+    similar than the texts of the first two files.  
+
+    Args:
+        filename_in (str): This parameter is the name of the file which is read.
+            The has to be in the directory /data.
+        filename_out (str): This parameter is the name of the file to which the
+            results are written. The file is created in the directory /data.
+
+    Returns:
+        None: This function does not return a value.
+    """
+    # The files that the cosine similarity is computed for has to be in 
+    # the directory testTextPais_WebForm.
+    path_texts = '../data/testTextPairs_WebForm/'
+    # Open the file that contains the filenames of the files that the cosine
+    # similarity is computed for and read it line by line.
+    file = open('../data/' + filename_in, "r")
+    lines = file.readlines()
+    file.close()
+    # Count the number of wrong predictions in filename_in.
+    wrong_predictions = 0
+
+    for i in range(0, len(lines)):
+        # Split each line at the space character to get the filenames.
+        lines[i] = lines[i].split(" ")
+        texts = []  # Content of the files is stored in this list.
+        for j in range(0, len(lines[i]) - 1):
```

Comment:
check for the right number of columns already here. 
If somebody has given a completely wrong file, it can lead to obscure error messages, like File Volkmar not found.